generators/entity/templates/src/main/webapp/app/entities/infinite-scroll-template.ejs

```diff
@@ -29,7 +29,11 @@
         <%_ } else { _%>
             if (vm.currentSearch) {
```

Comment:
what is the purpose of `vm.currentSearch`? its always same as `vm.searchQuery` can we remove the variable `currentSearch` altogether and just use `searchQuery` everywhere?